README.md

```diff
@@ -1,9 +1,21 @@
 frdomain-extras.kt
 ========
 
-Arrow port of the additional [Functional and Reactive Domain Modeling](https://www.manning.com/books/functional-and-reactive-domain-modeling) code samples. You can find the Scala version [here](https://github.com/debasishg/frdomain-extras).
+Arrow port of the additional [Functional and Reactive Domain Modeling](https://www.manning.com/books/functional-and-reactive-domain-modeling) code samples (Chapter 6). You can find the Scala version [here](https://github.com/debasishg/frdomain-extras).
+
+This project contains two implementations:
+
+- Implementation specialized with [Arrow's IO][arrow-io] effect
```

Comment:
IO and Fx? Link to the fx docs :D

Reply:
Link added to Arrow Fx (hope it's the right one)